components/HomeScreen/ContinentsList/ContinentsList.utils.js

```diff
@@ -2,16 +2,16 @@
  * creates a function to be called in the continent Pressable
  *
  * @param {object} args
- * @param {object} args.router
- * @param {function} args.router.push
+ * @param {object} args.navigation
```

Comment:
JS docs aren't just for declaring prop types and expected returns. They should also give detail about what the values actually are so that a user who didn't write the function can easily use it.

Reply:
for example I have no idea what kind of code line 7 expects.